Tools/packages.config

```diff
@@ -7,5 +7,5 @@
   <package id="NUnit.Extension.NUnitV2ResultWriter" version="3.6.0" targetFramework="net461" />
   <package id="NUnit.Extension.TeamCityEventListener" version="1.0.2" targetFramework="net461" />
   <package id="NUnit.Extension.VSProjectLoader" version="3.6.0" targetFramework="net461" />
-  <package id="CSharpAsyncGenerator.CommandLine" version="0.3.6" targetFramework="net461" />
+  <package id="CSharpAsyncGenerator.CommandLine" version="0.4.0" targetFramework="net461" />
```

Comment:
Required for newly generated tests, due to [#41](https://github.com/maca88/AsyncGenerator/issues/41), [#43](https://github.com/maca88/AsyncGenerator/issues/43), [#44](https://github.com/maca88/AsyncGenerator/issues/44) & [#45](https://github.com/maca88/AsyncGenerator/issues/45).